index.js

```diff
@@ -1,20 +1,45 @@
 const nodePath = require('path');
-const resolve = require('resolve');
+const regexIsCompat = /\.js$/;
+const regexIsRelative = /^\./;
 
 function webify(path) {
-  const specifier = path.node.source.value;
-  const filename = this.file.opts.filename;
-  const base = nodePath.dirname(
-    nodePath.join(process.cwd(), filename)
-  );
-
-  const pth = resolve.sync(specifier, {
-    basedir: base
-  });
-  const rel = nodePath.relative(base, pth);
-  const res = rel[0] === '.' ? rel : `./${rel}`;
-
-  path.node.source.value = res;
+  const { source } = path.node;
+  const { value } = source;
+  const isRelative = regexIsRelative.test(value);
+
+  // If it's web compatible, we simply let it pass through.
+  if (regexIsCompat.test(value)) {
```

Comment:
What if someone loads a library that ends in just like `import 'three.js'`.  What if someone uses `.mjs` as an extension. Or .jsx or .flow?  I think it's not safe to do this.  Instead we need to check if it is a bare module specifier and apply your special logic in that scenario.